thread/thread.cpp

```diff
@@ -1090,6 +1090,12 @@ R"(
         }
         return photon::now;
     }
+    void alog_update_now(uint64_t *sec, uint64_t *usec) {
```

Comment:
better return sec and usec directly as an std::pair, so that they are returned in registers.
```cpp
#include <stdio.h>
#include <inttypes.h>
#include <utility>

std::pair<uint64_t, uint64_t> foo();


int main(int argc, char** argv) {
	auto x = foo();
	return (x.first + x.second) >> 3;
}
```

sec/usec are returned in rax/rdx as a result:
```
(gdb) disassemble main
Dump of assembler code for function main:
   0x0000000000001050 <+0>:	sub    $0x8,%rsp
   0x0000000000001054 <+4>:	callq  0x1030 <_Z3foov@plt>
   0x0000000000001059 <+9>:	add    $0x8,%rsp
   0x000000000000105d <+13>:	add    %rdx,%rax
   0x0000000000001060 <+16>:	shr    $0x3,%rax
   0x0000000000001064 <+20>:	retq
End of assembler dump.
```

Reply:
updated: return `struct timeval` as result